big_tests/tests/metrics_c2s_SUITE.erl

```diff
@@ -267,3 +268,13 @@ error_iq(Config) ->
     Alice = escalus_users:get_user_by_name(alice, Users),
     escalus_users:create_user(Config, Alice),
     wait_for_counter(Errors + 1, xmppErrorIq).
+
+error_double_open_stream_must_close_connect(Config) ->
+    AliceSpecs = escalus_fresh:create_fresh_user(Config, alice),
+    Steps = [start_stream, stream_features],
+    {ok, Alice, _} = escalus_connection:start(AliceSpecs, Steps),
+    try
+        escalus_connection:start_stream(Alice)
+    catch error:"Not a valid stream start" ->
+        escalus_connection:wait_for_close(Alice, timer:seconds(5))
+    end.
```

Comment:
As @bartekgorny suggested, maybe `connect_SUITE` is a better suite to place this test. You can make a CT group that run its testcases in parallel, with all the testcases for all the steps of the FSM for example.